contract-tests/tests/test/amazon/base/contract_test_base.py

```diff
@@ -90,6 +90,7 @@ def setUp(self) -> None:
             .with_env("OTEL_METRIC_EXPORT_INTERVAL", "50")
             .with_env("OTEL_AWS_APPLICATION_SIGNALS_ENABLED", "true")
             .with_env("OTEL_METRICS_EXPORTER", "none")
+            .with_env("OTEL_EXPORTER_OTLP_PROTOCOL", "grpc")
```

Comment:
Note that this works because we install `opentelemetry-exporter-otlp-proto-grpc==1.22.0` in each sample application Docker image: https://github.com/search?q=repo%3Aaws-observability%2Faws-otel-python-instrumentation%20opentelemetry-exporter-otlp-proto-grpc%3D%3D1.22.0&type=code